lib/cpp/src/thrift/server/TThreadPoolServer.h

```diff
@@ -105,19 +105,16 @@ class TThreadPoolServer : public TServer {
       timeout_(0),
       taskExpiration_(0) {}
 
-  virtual ~TThreadPoolServer();
+  virtual ~TThreadPoolServer() {}
```

Comment:
This is nit-picky... but I would prefer you revert this particular change.  Please leave the virtual dtor out-of-lined.  An inlined virtual destructor can cause significant object size bloat.  This doesn't end up leaking into final binaries, but it can make build times and file copy times longer.